beacon-light-client/circom/scripts/common.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+
+look_for_ptau_file() {
+  if [ -f "$PHASE1" ]; then
+    echo "Found Phase 1 ptau file"
+  else
+    echo "No Phase 1 ptau file found. Exiting..."
+    exit 1
+  fi
+}
+
+create_build_folder() {
+  if [ ! -d "$BUILD_DIR" ]; then
+    echo "No build directory found. Creating build directory..."
+    mkdir -p "$BUILD_DIR"
+  fi
+}
+compile_the_circuit() {
+  echo "****COMPILING CIRCUIT****"
+  start=$(date +%s)
+  cd "${CIRCUIT_DIR}" || exit
+  #circom "$CIRCUIT_NAME".circom --O0 --c --output "$BUILD_DIR"
+  circom "${CIRCUIT_NAME}".circom --O2 --r1cs --sym --c --output "$BUILD_DIR"
+  end=$(date +%s)
+  echo "DONE ($((end - start))s)"
+}
```

Comment:
Same suggestion here.
```suggestion
compile_the_circuit() {
  local circuit_dir="$1"
  local circuit_name="$2"
  
  echo "****COMPILING CIRCUIT****"
  start=$(date +%s)
  cd "$circuit_dir" || exit
  #circom "$circuit_name".circom --O0 --c --output "$BUILD_DIR"
  circom "$circuit_name".circom --O2 --r1cs --sym --c --output "$BUILD_DIR"
  end=$(date +%s)
  echo "DONE ($((end - start))s)"
}
```
Then you can call it like follows:
```sh
compile_the_circuit "$CIRCUIT_DIR" "$CIRCUIT_NAME"
```

Reply:
Done